widget/embedded/src/hooks/useSyncStoresWithConfig.ts

```diff
@@ -90,30 +82,21 @@ export function useSyncStoresWithConfig() {
   ]);
 
   useEffect(() => {
-    if (fromToken && fromTokensConfig) {
-      if (!fromTokensConfig.some((token) => tokensAreEqual(token, fromToken))) {
-        setFromToken({ token: null });
-      }
+    if (shouldChangeSelectedToken(fromToken, fromTokensConfig)) {
```

Comment:
Maybe `reset` makes more sense than `change` in this context. (for both functions)

Reply:
fixed